dev/devicelab/bin/tasks/plugin_test_ios.dart

```diff
@@ -13,6 +13,9 @@ Future<void> main() async {
     // Test that Dart-only plugins are supported.
     PluginTest('ios', <String>['--platforms=ios'], dartOnlyPlugin: true).call,
     PluginTest('macos', <String>['--platforms=macos'], dartOnlyPlugin: true).call,
+    // Test that shared darwin directories are supported.
+    PluginTest('ios', <String>['--platforms=ios,macos'], sharedDarwinSource: true).call,
+    PluginTest('macos', <String>['--platforms=ios,macos'], sharedDarwinSource: true).call,
```

Comment:
This test is now taking more than 30 minutes.  Will split it up.

Reply:
Hm, it's still timing out...

Reply:
Looks like the new unit tests I added to the template are working :)

It's probably easiest to ifdef the `result` line that you're already replacing to match the existing implementations than it is to have it rewrite the unit test files as well.